autoload/airline/extensions/tabline/ctrlspace.vim

```diff
@@ -0,0 +1,97 @@
+" MIT License. Copyright (c) 2016 Kevin Sapper
+" vim: et ts=2 sts=2 sw=2
+
+scriptencoding utf-8
+
+let s:current_bufnr = -1
+let s:current_tabnr = -1
+let s:current_tabline = ''
+
+function! airline#extensions#tabline#ctrlspace#off()
+  augroup airline_tabline_ctrlspace
+    autocmd!
+  augroup END
+endfunction
+
+function! airline#extensions#tabline#ctrlspace#on()
+  augroup airline_tabline_ctrlspace
+    autocmd!
+    autocmd BufDelete * call airline#extensions#tabline#ctrlspace#invalidate()
+  augroup END
+endfunction
+
+function! airline#extensions#tabline#ctrlspace#invalidate()
+  let s:current_bufnr = -1
+  let s:current_tabnr = -1
+endfunction
+
+function! airline#extensions#tabline#ctrlspace#get()
+  let cur_buf = bufnr('%')
+
+  let s:tab_list = ctrlspace#api#TabList()
+  for tab in s:tab_list
+    if tab.current
+      let cur_tab = tab.index
+    endif
+  endfor
+
+  if cur_buf == s:current_bufnr && cur_tab == s:current_tabnr
+    return s:current_tabline
+  endif
+
+  let b = airline#extensions#tabline#new_builder()
+
+  call b.add_section_spaced('airline_tabtype', 'buffers')
```

Comment:
Hey @chrisbra, the integration with ctrlspace looks great, loving it after tried out for a few days.

Can we add options to customize the text on the airline? I personally would prefer to replace `buffers` and `tabs` to some symbol texts to give more space on the tabline.

Also if we can have an options like this:

`let g:airline#extensions#tabline#reverse_buffers_and_tabs = 1`

it will gave users more flexibility to have tabs on the left and buffers on the right.

What do you think ?

Also cc @bling